src/main/java/hypersquare/hypersquare/dev/code/player/action/PlayerSetInventoryItems.java

```diff
@@ -0,0 +1,122 @@
+package hypersquare.hypersquare.dev.code.player.action;
+
+import hypersquare.hypersquare.dev.BarrelParameter;
+import hypersquare.hypersquare.dev.BarrelTag;
+import hypersquare.hypersquare.dev.action.Action;
+import hypersquare.hypersquare.dev.codefile.data.CodeActionData;
+import hypersquare.hypersquare.item.action.ActionItem;
+import hypersquare.hypersquare.item.action.ActionMenuItem;
+import hypersquare.hypersquare.item.action.player.PlayerActionItems;
+import hypersquare.hypersquare.item.value.DisplayValue;
+import hypersquare.hypersquare.menu.barrel.BarrelMenu;
+import hypersquare.hypersquare.play.CodeSelection;
+import hypersquare.hypersquare.play.execution.ExecutionContext;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+// For player inventories! Not inventory menus.
```

Comment:
minor, but comment is unneeded and looks bad (menu would be called PlayerSetInventoryMenuItems)